libs/designsystem/card/src/card.component.scss

```
@@ -46,30 +46,22 @@
   z-index: utils.z('default');
   width: var(--kirby-card-width, auto);
 
-  .content-wrapper {
-    &.padding {
-      padding-top: var(--kirby-card-padding-top, utils.size('s'));
-      padding-bottom: var(--kirby-card-padding-bottom, utils.size('s'));
-      padding-inline: utils.size('s');
-    }
-  }
-
   &.flat {
     box-shadow: none;
   }
 
   &.outlined {
     background-color: transparent;
-    border: 1px solid utils.get-color('medium');
+    border: var(--kirby-card-border, 1px solid utils.get-color('medium'));
     box-shadow: none;
 
     &:has(kirby-card-header.flagged) {
       border: none;
 
       .content-wrapper {
-        border-block-end: 1px solid utils.get-color('medium');
-        border-inline-start: 1px solid utils.get-color('medium');
-        border-inline-end: 1px solid utils.get-color('medium');
+        border-block-end: var(--kirby-card-border, 1px solid utils.get-color('medium'));
+        border-inline-start: var(--kirby-card-border, 1px solid utils.get-color('medium'));
+        border-inline-end: var(--kirby-card-border, 1px solid utils.get-color('medium'));
```

Consider extracting to Sass variable:
```suggestion
        $border-outline: 1px solid utils.get-color('medium')-,
        border-block-end: var(--kirby-card-border, $border-outline);
        border-inline-start: var(--kirby-card-border, $border-outline);
        border-inline-end: var(--kirby-card-border, $border-outline);
```